config/schedule.rb

```diff
@@ -38,6 +38,7 @@
   rake 'deny_missed_requests'
   rake 'delete_old_blackouts'
   rake 'email_missed_reservations'
+  rake 'email_checkout_reminder'
```

Comment:
you should also update the `scheduler` task for Heroku, now that that's been merged into `master`.